acct-user/quassel/quassel-0.ebuild

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+ACCT_USER_ID=472
+ACCT_USER_HOME=/var/lib/quassel
```

Comment:
Do you need this, given that init.d hardcodes the path anyway?

Reply:
I just migrated the existing function to create the cert of pkg net-irc/quassel

Reply:
I don't know how this is relevant. FWICS, the function also uses hardcoded path.

Reply:
note that I haven't done a GLEP81 ebuild so far; isn't `/var/lib/<username>` the default anyway, and it is recommendet not to be set in that case?

Reply:
`/dev/null` is the default, and it's recommended to leave it as that.

Reply:
Where is the difference between acct-user/git with  www-apps/gitea for example to quassel? So should I use a designed eclass function to create the dir with the correct user and group or should I do it in ebuild?